server/src/main/java/au/org/aodn/ogcapi/server/core/service/OGCApiService.java

```diff
@@ -78,11 +79,20 @@ public String processDatetimeParameter(String datetime, String filter) {
 
         // for now, assumption is that temporal is the only filter
         if (filter == null) {
-            if (datetime.startsWith("../") || datetime.startsWith("/")) {
+            if (datetime.startsWith("../")) {
```

Comment:
Maybe I am wrong, looks like the cql do support ../datetime  datetime/datetime. if that is the case, then you can pass the datetime value directly

Reply:
Also I try this query localhost:8080/collections?filter=temporal during 2008-07-16T00:00:00Z/2014-11-10T00:00:00.000Z

this record "9aa6fcf6-7da0-431b-897e-8278fa6f5161" get return where time is 
            "temporal": [
              [
                "2013-05-01T00:00:00+10:00",
                "2015-03-28T00:00:00+11:00"
              ]
            ]

which seems not right?

Reply:
Something wrong with using nested query

https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-nested-query.html

Reply:
which code you are in? I do the same query as you and get this response instead:

![image](https://github.com/aodn/ogcapi-java/assets/26201635/b1bca318-671b-45b7-b678-1b0ae8e15bbe)

the temporal looks right with the hit results:

![image](https://github.com/aodn/ogcapi-java/assets/26201635/81904e8b-7b70-4c3d-b828-febf4e08a8c9)
 @utas-raymondng

Reply:
the end time is in 2015, but **2013-04-30 to 2014-11-10** still fall into the covering period I guess?

Reply:
![fff drawio (1)](https://github.com/aodn/ogcapi-java/assets/26201635/5a828f6b-b212-4517-b594-1044705f23e1)
@utas-raymondng

Reply:
> Maybe I am wrong, looks like the cql do support ../datetime datetime/datetime. if that is the case, then you can pass the datetime value directly

I've refactored and add a debug logger to verify what CQL expression being used for returning results. also please check to TODO note with case end with "/" and "datetime/datetime" (should be datetime1/datetime2 - different timestamps - to be naturally valid)